apps/token-manager/app/src/components/InfoBoxes.js

```diff
@@ -110,13 +111,17 @@ function InfoBoxes({
           items={stakes}
           renderLegendItem={({ item: account }) => {
             const isCurrentUser = addressesEqual(account, connectedAccount)
+            const [label, showLocalIdentityModal] = useIdentity(account)
+            
             return (
               <div>
                 <LocalIdentityBadge
                   entity={account}
                   connectedAccount={isCurrentUser}
+                  customLabel={!Boolean(label) && isCurrentUser && 'YOU'}
+                  labelStyle={(!Boolean(label) && isCurrentUser ? `color: ${theme.tagIndicatorContent};` : '')}
```

Comment:
I'm wondering—maybe we should move this logic into the `LocalIdentityBadge`.

We already provide a prop via `connectedAccount` to control whether or not it's "you", and we can detect and build off of this in that component. We can leave a prop to enable this "special" connected mode, to default the label and style.

Reply:
I didnt wanted to do it in `LocalIdentityBadge` cause I thought this change was only going to be used here because we dont have much space.

Reply:
Yes... I was thinking it could be a special mode of the `LocalIdentityBadge` but this falls apart with needing the label to detect whether we should display the "you".